TCC/Parser/BenchmarkOptions.cs

```diff
@@ -12,23 +13,34 @@ public class BenchmarkOptions
         [Option('a', "algorithm", HelpText = "Algorithm : All (default), Lz4, Brotli, Zstd", Default = BenchmarkCompressionAlgo.All)]
         public BenchmarkCompressionAlgo Algorithm { get; set; }
 
-        [Option('r', "ratio", HelpText = "Compression ratio. Valid values are : lz4 [1,9], brotli [1,11], zstd [1,19]. No value benchmarks all ratios ")]
+        [Option('r', "ratio", HelpText = "Compression ratio. Valid values are : lz4 [1,9], brotli [1,11], zstd [1,19] (ex : '3'). No value benchmarks all ratios !")]
         public int Ratio { get; set; }
 
-        [Option('e', "encrypt", HelpText = "With or without encryption. Both tested by default", Default = null)]
+        [Option('e', "encrypt", HelpText = "With or without encryption. Valid values are : 'true','false'. Both tested by default", Default = null)]
         public bool? Encrypt { get; set; }
 
-        [Option('c', "content", HelpText = "Ascii, Binary, Both. If no specified source, generated test file content. Both by default", Default = BenchmarkContent.Both)]
+        [Option('c', "content", HelpText = "Ascii, Binary, Both. Default = Both. (use only if explicit source parameter)", Default = BenchmarkContent.Both)]
         public BenchmarkContent Content { get; set; }
 
-        [Option('n', "number", HelpText = "Number of generated test files. Default = 1", Default = 1)]
+        [Option('n', "number", HelpText = "Number of generated test files. Default = 1. (use only if explicit source parameter)", Default = 1)]
         public int NumberOfFiles { get; set; }
 
-        [Option('s', "size", HelpText = "Size of generated test files in Kb. Default = 102 400 (100Mb)", Default = 102400)]
+        [Option('s', "size", HelpText = "Size of generated test files in Kb. Default = 102 400 (100Mb). (use only if explicit source parameter)", Default = 102400)]
         public int FileSize { get; set; }
 
         [Option('t', "threads", HelpText = "Number of parallel threads. Default = Processor logical core count", Default = 0)]
         public int Threads { get; set; }
+
+        [Option('o', "outputCompressed", HelpText = "Output folder for compressed archives. Default = user temp folder", Default = null)]
+        public string OutputCompressed { get; set; }
+
+        [Option('d', "outputDecompressed", HelpText = "Output folder for archive decompression. Default = user temp folder", Default = null)]
+        public string OutputDecompressed { get; set; }
+
+        [Option('x', "cleanup", HelpText = "Cleanup compressed / decompressed data between each pass. Valid values are : 'true','false'", Default = false)]
+        public bool Cleanup { get; set; }
+
+        internal static HashSet<string> AutoTestDataOptions = new HashSet<string>() { "-c", "--content", "-n", "--number", "-s", "--size" };
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Change the visibility of 'AutoTestDataOptions' or make it 'const' or 'readonly'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS2223)